chapters/api_concepts.txt

```diff
@@ -247,18 +247,31 @@ typedef struct
   ANARIDataType type;
 } ANARIParameter;
 
-const ANARIParameter *anariGetObjectParameters(ANARILibrary,
+const void *anariGetObjectInfo(ANARILibrary,
     const char *deviceSubtype,
     ANARIDataType objectType,
-    const char *objectSubtype);
+    const char *objectSubtype
+    const char *infoName,
+    ANARIDataType infoType);
 ----
 
-An object (sub)type `objectType`/`objectSubtype` can be queried for supported
-parameters with `anariGetObjectParameters` (passing `NULL` or an empty string
+An object (sub)type `objectType`/`objectSubtype` can be queried for information
+with `anariGetObjectInfo` (passing `NULL` or an empty string
 for `objectSubtype` to query an object type directly that does not have
-subtypes). The function returns `NULL` if there are no parameters, or an array
-of `ANARIParameter` structs, where the last element has `NULL` as name and
-`ANARI_UNKNOWN` as type.
+subtypes). The function returns the result for `infoName` of type `infoType`, or
+`NULL` if the information can not be retrieved.
```

Comment:
Can we change to the more formal "cannot" (can not) and "information" (info) in running text?